### ISSUE TITLE
New Dashboard UX

### ISSUE THREAD

**Issue by gemfarmer** [18F/micropurchase#525]
@mtorres253 @andrewmaier brain dump on ways to improve the Dashboard page and related pages.

Currently the UX for the [winners page dashboard](https://micropurchase.18f.gov/auctions/winners/) has a cluttered focus. The original issue to create the dashboard ([here](https://trello.com/c/DvU4RlrH/119-previous-winners-page)) names 3 user stories:

> As a **government oversight official**, I want to
> - see how the government is using the platform
> - see what kind of results the government is receiving
> - see if(when) there are failures
> - see how money is being spent and on what
> - understand how GSA is avoiding split purchases

> As a **potential bidder (vendor)**, I need to
> - be able to see what kind of auctions have been posted in the past
> - what vendors have won in the past
> - see how much previous auctions cost
> - see links to GitHub code (PR's) for work that was delivered

> As a **government employee (customer)**, I want to
> - know what kind of work has been delivered on the platform
> - see how much previous agencies have paid
> - see one page of all "non-active" archived opportunities

While the current dashboard accomplishes some of this, it definitely needs enhancements before it is truly useful to those users. When @mtorres253 and I spoke the other day, we discussed the potential to narrow our scope and to focus on implementations that have measurable utility. 

The new scope would focus on creating pages and visualizations that would enhance our (the micropurchase team) understanding of what is going on in the budding micropuchase ecosystem. Here are a few things that we discussed/brainstormed, extrapolating from this [Google Doc](https://docs.google.com/document/d/1Msi0rvhB1vSZsn4m9Td_vIHtyq_F154OOzJ6kud-_bI/edit):

Our new user story [WIP]:
> As a **micropurchase team member**, I want to
> - be able to see general trends in the micrpurchase platform
> - see analysis and visualizations on a per-repo basis to better understand how auctions are impacting a particular project.
> - see trends for a particular bidder (vendor). 
> - see high-level trends for different auction types, to better inform our auction-type recommendations for new projects

What this might mean for development
- **Repo pages**: 
  - high level pages that contain info about a particular project/repo
    - list of auctions for that repo
    - average award for that repo
    - average auction length for that repo
    - languages and stacks uses
    - payment info
  - sync with existing auction pages
- **Auction pages**
  - add visualizations to existing auction pages
- **Vendor pages**: 
  - page specific to a given vendor
  - bidding history
  - successful contributions (would play into the idea of reputation)
  - would help track nefarious vendors, help us and potential client leads determine if they are working with a reliable vendor.
- **Enhancements within existing page**: 
  - links to auctions within visualizations
- **Measurables**
  - The Community chart in the dashboard is currently a bit clunky. This is partially because we have so few auctions that the time series axis is empty, making the "trend" virtually non-existent. When this chart becomes more visually appealing and useful (basically more frequent auctions), we will know that Micropurchase is succeeding.

**Comment by mtorres253** [comment]
I would add some kind of time filter. I'd like to be able to compare data in the beginning vs. recently

**Comment by gemfarmer** [comment]
Definitely. We could default to only showing data in the last few months, and allow people to choose to look back in history.
